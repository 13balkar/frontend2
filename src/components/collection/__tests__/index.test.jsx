import * as React from 'react';
import { render } from '@testing-library/react';
import Collection from '../';
jest.mock('react-router-dom', () => ({
  useNavigate: () => jest.fn()
}));

describe('Collection', () => {
  it('should render the collection', () => {
    const { asFragment } = render(<Collection />);
    expect(asFragment()).toMatchSnapshot();
  });
});
