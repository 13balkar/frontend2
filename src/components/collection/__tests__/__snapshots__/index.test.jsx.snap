// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Collection should render the collection 1`] = `
<DocumentFragment>
  <div
    class="collection"
  >
    <div
      class="collection-name"
    >
      <h1 />
    </div>
  </div>
</DocumentFragment>
`;
